src/napari_spatialdata/_widgets.py

```diff
@@ -379,6 +493,34 @@ def cmap(self) -> str:
         return self._cmap
 
 
+class MatplotlibWidget(NapariMPLWidget):
+    def __init__(self, viewer: Viewer, model: ImageModel):
+
+        super().__init__(viewer)
+
+        self.viewer = viewer
+        self.model = model
+        self.axes = self.canvas.figure.subplots()
+
+    def clear(self) -> None:
+        self.axes.clear()
```

Comment:
general design principle, put methods/attributes from the most complex to the least complex in a class. This one for isntance it's just one line of code, so should be put at the end of the class, whereas more complex methods (either in terms of quantity of code, or diffficulty of logic) should be put on top.